include/staff/orgs.inc.php

```diff
@@ -219,10 +219,10 @@
             $form.find('#selected-count').val(ids.length);
             $form.submit();
           };
-          $.confirm(__('You sure?')).then(function(promise) {
-            if (promise === false)
+          $.confirm(__('You sure?')).then(function(data) {
+            if (data === false)
               return false;
-            submit();
+            submit(data);
```

Comment:
Submit inline-function doesn't support $data - wonder if the change was necessary here.